src/components/PostsList.tsx

```diff
@@ -87,0 +1,120 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Post } from '../types/Post';
+import classnames from 'classnames';
+import { getPostsByUserId } from '../api/posts';
+import { User } from '../types/User';
+import { Loader } from './Loader';
+
+type Props = {
+  selectedUser: User | null;
+  onPostSelect: (post: Post | null) => void;
+};
+
+export const PostsList: React.FC<Props> = ({ onPostSelect, selectedUser }) => {
+  const [posts, setPosts] = useState<Post[]>([]);
+  const [selectedPost, setSelectedPost] = useState<Post | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
The variable name `isLoad` is not clear. Consider renaming it to `isLoading` or `isLoaded` to make the code more readable and understandable.